src/gameobjects/components/Core.js

```diff
@@ -361,7 +361,10 @@ Phaser.Component.Core.prototype = {
         {
             var child = this.children[i];
 
-            child.preUpdate();
+            if (child.active)
```

Comment:
Samesies

Reply:
I'm getting too lazy to keep marking these, so just anywhere that we're doing this.

Reply:
I removed all the preUpdate child checks, but I kept in the render and postUpdate child checks.